src/examples/forms/range/RangeDefault.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, Range } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  return (
+    <Row>
+      <Col>
```

Comment:
Now that this has been simplified, it's clear that `<Row>` and `<Col>` are not necessary in any of these examples.